SYCL/ESIMD/regression/double_conversion_test.cpp

```diff
@@ -0,0 +1,82 @@
+//==--- double_conversion_test.cpp - DPC++ ESIMD on-device test ---------==//
+//
+// Part of the LLVM Project, under the Apache License v2.0 with LLVM Exceptions.
+// See https://llvm.org/LICENSE.txt for license information.
+// SPDX-License-Identifier: Apache-2.0 WITH LLVM-exception
+//
+//===----------------------------------------------------------------------===//
+// This is basic test for testing conversion between double and unsigned int.
+
+// REQUIRES: gpu
+// UNSUPPORTED: cuda || hip
+// RUN: %clangxx -fsycl %s -o %t.out
+// RUN: %GPU_RUN_PLACEHOLDER %t.out
+
+#include <sycl/ext/intel/esimd.hpp>
+#include <sycl/sycl.hpp>
+
+#include <ios>
+#include <iostream>
+#include <typeinfo>
+
+namespace esimd = sycl::ext::intel::esimd;
+
+int test(sycl::queue queue, double testValue) {
```

Comment:
Accordingly to clang naming/format rules 'testValue' would be a good name for a function: testValue().
The vars and types start with capital letter, i.e.: 'TestValue'.